src/components/search-input/search-input.stories.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./../../styles.scss";
+import { withKnobs, boolean, text } from "@storybook/addon-knobs";
+import { SearchInput } from "./search-input";
+
+export default { title: "Search Input", decorators: [withKnobs] };
+
+export const searchInput = () => <SearchInput></SearchInput>;
+export const icon = () => (
+  <SearchInput hasIcon={boolean("Has search", true)}></SearchInput>
```

Comment:
Considering that our default value for `hasIcon: true`, would it make sense to actually have this story, or maybe another one with `hasIcon: false`?

Reply:
We have the knobs to change the value, but sure, it would be more relevant to have the `hasIcon: false` for this story :)

Reply:
You are completely right. My bad. Maybe change the default to `false` in `knobs`? Otherwise, it's fine the way it is. I just missed the `knob`